TODO: I need to look into if this is valid, right now I just want oniux to be
usable again

--- a/src/main.rs
+++ b/src/main.rs
@@ -222,7 +222,8 @@ fn main_main(args: Args) -> Result<ExitC
     }
 
     if !user::usable() {
-        bail!("user namespaces not usable");
+    //  bail!("user namespaces not usable");
+    //  not available
     }
 
     // Create IPC primitives.
